llvm/utils/TableGen/SubtargetEmitter.cpp

```diff
@@ -317,18 +308,17 @@ SubtargetEmitter::CPUKeyValues(raw_ostream &OS,
       Records.getAllDerivedDefinitions("Processor");
   llvm::sort(ProcessorList, LessRecordFieldName());
```

Comment:
Check for duplicated here, too?

Reply:
Yeah, its already checked by the time this code executes. SubtargetEmitter builds a CodeGenSchedModels object in the constructor:

```
  SubtargetEmitter(RecordKeeper &R)
      : TGT(R), Records(R), SchedModels(TGT.getSchedModels()),
        Target(TGT.getName()) {}
```

which checks for duplicate Processor. I'll add a comment to that effect in `CPUKeyValues`.

Reply:
Added a comment.